src/autorest-core/resources/default-configuration.md

```diff
@@ -111,14 +111,12 @@ pipeline:
     input: individual/transform
     scope: azure-validator-individual
   # validator written in TypeScript
-  swagger-document/azure-openapi-validator:
-    input:
-      - transform
-      - azure-validator # artificial predecessor in order to ensure order of messages for CI purposes
-  swagger-document/individual/azure-openapi-validator:
-    input: 
-      - transform
-      - azure-validator # artificial predecessor in order to ensure order of messages for CI purposes
+  swagger-document/azure-openapi-linter:
+    input: transform
+    scope: azure-validator-composed # artificial predecessor in order to ensure order of messages for CI purposes
```

Comment:
the comment `# artificial predecessor in order to ensure order of messages for CI purposes` makes no sense here anymore! it was meant as an explanation for the 2nd input, so it doesn't accidentally get removed... but it is removed now, and the comment was just moved to a place where it has no meaning. As the comment is trying to clarify, the 2nd input is imposing a dependency which ensures *determinism* of our output! Unless CI and tests are updated to not rely on order of emitted messages, this will only pass CI with extreme luck.

Personally, I also like programs that produce the same output if run twice ;)